hrvisa/info.xml

```diff
@@ -10,11 +10,10 @@
   </maintainer>
   <releaseDate>2017-08-22</releaseDate>
```

Comment:
Hope this `releaseDate` date is correct as it was not updated?

Reply:
same thing as hrident

Reply:
Alright @davialexandre